components/LocationPreviewCard/index.js

```diff
@@ -64,12 +65,17 @@ margin: 0;
 width = 30%;
 align-self: flex-end;
 align: right`;
-
+const NavLink = styled(Link)`
+  text-decoration: none;
+  &: hover {
+    font-size: 1.2em;
+  }
+`;
 export default function LocationPreviewCard({ name, addresse, bild, id }) {
   return (
     <>
```

Comment:
We don't need the `React Fragment` in this case, seeing as we are only returning one element from this component (`StyledListItem`).